DemoBoolean.java

```diff
@@ -0,0 +1,11 @@
+package com.edocti.jintro.lab01;
+	
+public class DemoBoolean{
```

Comment:
General comment: you can use a styling guide. Google has a good one: https://google.github.io/styleguide/javaguide.html